<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>380</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximumSize">
     <size>
      <width>380</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Inconsolata</family>
      <pointsize>35</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 229, 255);
qproperty-alignment: AlignRight;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>381</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_0">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="pushButton_1">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_7">
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_8">
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="pushButton_9">
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButton_sign">
       <property name="text">
        <string>±</string>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QPushButton" name="pushButton_add">
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QPushButton" name="pushButton_sub">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="6">
      <widget class="QPushButton" name="pushButton_mul">
       <property name="text">
        <string>×</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <widget class="QPushButton" name="pushButton_div">
       <property name="text">
        <string>÷</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButton_sqrt">
       <property name="text">
        <string>√</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButton_sqr">
       <property name="text">
        <string>x²</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_clear">
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QPushButton" name="pushButton_dot">
       <property name="text">
        <string>.</string>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_equal">
       <property name="font">
        <font>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>=</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="5">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionN_ew">
   <property name="text">
    <string>N&amp;ew</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
